framework/py/flwr/supercore/superexec/run_superexec.py

```diff
@@ -120,3 +126,44 @@ def get_run(run_id: int) -> Run:
             time.sleep(1)
     finally:
         channel.close()
+
+
+def run_with_deprecation_warning(  # pylint: disable=R0913, R0917
+    cmd: str,
+    plugin_type: str,
+    plugin_class: type[ExecPlugin],
+    stub_class: Union[
+        type[ClientAppIoStub], type[ServerAppIoStub], type[SimulationIoStub]
+    ],
+    appio_api_address: str,
+    flwr_dir: Optional[str],
+    parent_pid: Optional[int],
+) -> None:
+    """Log a deprecation warning and run the equivalent `flower-superexec` command.
+
+    Used for legacy long-running `flwr-*` commands (i.e., without `--token`) that will
+    be removed in favor of `flower-superexec`.
+    """
+    log(
+        WARN,
+        "Running `%s` as a long-running process without `--token` is "
```

Comment:
```suggestion
        "Directly executing `%s` is "
```

I wonder if we should skip mentioning the "without `--token`". with the recent change a user won't know what token to pass right? If so, should we make the change as above?

Reply:
Yeah, good point. You are right that this may make users think they need to provide `--token`.